src/modules/service-catalog/components/service-catalog-list-item/EmptyState.tsx

```diff
@@ -0,0 +1,63 @@
+import { Anchor, Button } from "@zendeskgarden/react-buttons";
+import { LG, MD } from "@zendeskgarden/react-typography";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+
+const Container = styled.div`
+  padding: ${(p) => p.theme.space.xl} 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: ${(props) => props.theme.space.md};
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: ${(props) => props.theme.space.xxs};
+`;
+
+const StyledLink = styled.a`
+  &:visited,
+  &:hover {
+    text-decoration: none;
+    color: ${(props) => getColorV8("grey", 100, props.theme)};
+  }
+`;
+
+export const EmptyState = () => {
+  const { t } = useTranslation();
+  return (
+    <Container>
+      <TextContainer>
+        <LG>
+          {t("service-catalog.empty-state.no-services", "No services in sight")}
+        </LG>
+        <MD>
+          {t(
+            "service-catalog.empty-state.description",
+            "Once services are added to catalog, you’ll find them here."
+          )}
+        </MD>
+        <Anchor isExternal href="https://garden.zendesk.com/" target="_blank">
+          {/* Link will be updated when we will have support article */}
+          {t(
+            "service-catalog.empty-state.support-article-link",
+            "Learn about the service catalog"
+          )}
+        </Anchor>
```

Comment:
I would not add a link to the Garden website.

Either we just omit this link entirely until we have the support article or we just link to `#`

Reply:
I added `#` for now.